crates/turborepo-lib/src/run/cache.rs

```diff
@@ -16,6 +16,20 @@ use crate::{
     task_graph::{TaskDefinition, TaskOutputs},
 };
 
+#[derive(Debug, thiserror::Error)]
+pub enum Error {
+    #[error("Error replaying logs: {0}")]
+    Ui(#[from] turborepo_ui::Error),
+    #[error("Error accessing cache: {0}")]
+    Cache(#[from] turborepo_cache::CacheError),
+}
+
+impl Error {
+    pub fn is_cache_miss(&self) -> bool {
+        matches!(&self, Self::Cache(CacheError::CacheMiss))
+    }
+}
```

Comment:
Unused? (If you have a plan for use seems fine.)

Reply:
Yes, currently unused but will be to differentiate between a cache miss and an error accessing the cache which is needed [in run](https://github.com/vercel/turbo/blob/main/cli/internal/run/real_run.go#L443)